cPP_DSC.adoc

```diff
@@ -967,241 +989,317 @@ Extended SFRs (i.e. those SFRs that are not defined in [CC2]) are identified by
+|NIST SP 800-56B Rev 2 (Section 8.3)
+
+|DH
+|Diffie-Hellman
+|{empty}[*selection*: [.underline]#2048, 3072, 4096, 6144, 8192]# bits
+|NIST SP 800-56A Rev 3, {empty}[*selection*: RFC 3526 (Section [.underline]#[*selection*: 3, 4, 5, 6, 7]), RFC 7919 (Appendixes [*selection*: A.1, A.2, A.3, A.4, A.5]#)]
+
+|ECDH-NIST
+|ECDH with NIST curves
+|{empty}[*selection*: [.underline]#256 (P-256), 384 (P-384), 512 (P-521)]#
+|NIST SP 800-56A
+
+|ECDH-BPC
+|ECDH with Brainpool curves
+|{empty}[*selection*: [.underline]#256 (brainpoolP256r1), 384 (brainpoolP384r1), 512 (brainpoolP512r1)]#
+|RFC 5639 (Section 3)
+
+|ECDH-Ed
+|ECDH with Edwards Curves
+|{empty}[*selection*: [.underline]#256, 448]# bits
+|RFC 7748
 
-_Application Note {counter:remark_count}_:: _This is a refinement of the SFR FCS_CKM.2 to deal with key establishment rather than key distribution._
+|ECIES
+|ECIES
+|{empty}[*selection*: [.underline]#256, 384, 512]# bits
+|{empty}[*selection*: [.underline]#ANSI X9.63, IEEE 1363a, ISO/IEC 18033-2 Part 2, SECG SEC1 sec 5.1]#
+
+|KDF
+|{empty}[*selection*: [.underline]#KDF-CTR, KDF-FB, KDF-DPI]# with concatenated keys as input using {empty}[*selection*: [.underline]#AES-128-CMAC; AES-192-CMAC; AES-256-CMAC, HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as the PRF.
+|{empty}[*selection*: [.underline]#128, 192, 256]# bits
+|NIST SP 800-108 (Section 5) [KDF]
+
+{empty}[*selection*: [.underline]#ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC), ISO/IEC 18033-3:2010 (AES), ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC), ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)]# 
+
+|KEK
+|Encrypting one key with another
+|{empty}[*selection*: [.underline]#128, 192, 256]# bits
+|N/A
+
+|XOR
+|exclusive OR (XOR)
+|{empty}[*selection*: [.underline]#128, 192, 256]# bits
+|N/A
+
+|===
+
+_Application Note {counter:remark_count}_:: _This SFR captures methods for multi-party key agreement in which multiple parties contribute material used to derive the shared key used by each party to encrypt and decrypt messages to and from each other. TOEs can use the derived keys as symmetric keys, keyed-hash keys, or cryptographic keys for key derivation functions._
++
+_FCS_CKM.5 defines KDF-CTR, KDF-FB, and KDF-DPI._
 +
-_The ST author selects all key establishment schemes used for the selected cryptographic protocols._
+_For the KDF functions, when concatenating keys for AES-CMAC, the contributions from each party should be an equal number of bits, resulting in the selected key size (e.g., if each share is128 bits, then the result after concatenation is a 256-bit key, which is appropriate only for AES-256-CMAC). For HMAC functions, the shares can be any size as long as the concatenated result is at least equal to or greater than the nominal cryptographic strength of the chosen hash function (e.g. if each share is 128 bits, then the result after concatenation is 256 bits, which can be used in any of SHA-1, SHA-256, or SHA-512)._
```

Comment:
small typo: is128

Reply:
resolved. probably is an error in the catalog since that should have been a copy-paste.